test/runner/lib/powershell_import_analysis.py

```diff
@@ -62,7 +82,7 @@ def extract_powershell_module_utils_imports(path, module_utils):
 
         for line in lines:
             line_number += 1
-            match = re.search(r'(?i)^#\s*requires\s+-module(?:s?)\s*(Ansible\.ModuleUtils\..+)', line)
+            match = re.search(r'(?i)^#\s*(?:requires\s+-module|ansiblerequires\s+-powershell)(?:s?)\s*((?:Ansible|AnsibleCollections)\..+)', line)
```

Comment:
you have an option `s` after either `-module` or `-powershell`. Only module van be either `module` or `modules`.

https://github.com/ansible/ansible/blob/devel/lib/ansible/executor/powershell/module_manifest.py#L40-L41

Reply:
Good catch, that mistake was not intentional. I'll push a commit to correct that.